tools/scripts/utils.mjs

```diff
@@ -566,6 +611,39 @@ export const log = Object.assign(
   }
 );
 
+export const testThis = async (test) => {
```

Comment:
can we document this a bit more? its not inutative that this is wrapping the runtime tests.

Reply:
Agreed